Library/Account.cs

```diff
@@ -224,7 +225,8 @@ public void Update()
             // PUT /accounts/<account code>
             Client.Instance.PerformRequest(Client.HttpRequestMethod.Put,
                 UrlPrefix + Uri.EscapeDataString(AccountCode),
-                WriteXml);
+                WriteXml,
+                ReadXml);
```

Comment:
@naspthorp @chrissrogers in order to make this library work as the others when we perform the update(at least with account model) i had to pass the ReadXml so it overrides the the current state based on the latest info on the server. For example, if i update the invoice template uuid, the account response will include an href pointing to the new reference. 
I am honestly a little bit scare about doing this change

Reply:
Maybe i can create an special XMLReader that just update the invoice template _invoiceTemplateUuid private instance var